src/components/PersonLink.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { Sex } from '../types/Sex';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: React.FC<Props> = ({
+  person,
+}) => {
+  const {
+    slug, sex, name, born, died, mother, father, motherName, fatherName,
+  } = person;
+
+  const { slug: slugValue } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slugValue === slug,
```

Comment:
it doesn't work
![image](https://github.com/mate-academy/react_people-table-advanced/assets/86923089/10bbd0ee-dbd4-4eed-b457-2a8c1ff3d259)